data/data/bootstrap/files/usr/local/bin/wait-for-ha-api.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+wait_for_ha_api() {
+    if [ "$BOOTSTRAP_INPLACE" = false ]
+    then
+	echo "Waiting for at least 2 available IP addresses for default/kubernetes endpoint object"
+	while :
```

Comment:
Lets not do infinite loops and add `while (( attempts < max_attempts ))` instead

Reply:
we have the same in the configmap creation loop, is there a global timeout of this script maybe?

Reply:
oh, hmm, okay. The installer will cancel it after 30 mins anyway